.github/workflows/ci.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           server-token: ${{ env.TURBO_TOKEN }}
       
       - name: Version
-        run: yarn lerna version "$(node -e "console.log(require('semver').inc(require('./lerna.json').version, 'patch'))")-pre.${{ github.run_number }}" --no-git-tag-version --no-push --yes
+        run: yarn lerna version "$(node -e "console.log(require('semver').inc(require('./package.json').version, 'patch'))")-pre.${{ github.run_number }}" --no-git-tag-version --no-push --yes
```

Comment:
Missed this in the previous PR and had wired one to fix this as well.

Reply:
No worries, i will close this